docs/modules/ROOT/pages/performance/sql_and_scan_query_pref_comp.adoc

```diff
@@ -33,17 +33,17 @@ instantiation, cache.withKeepBinary().query(...)
 SCAN_CURSOR_FETCH time represents time required to extract the result from
 result set (cursor, cursor.getAll() API).
 
-For Insert operation, MAX represents total time to insert 1M entires into the cache and AVG represents the average time to insert one single entry. All other statistcs for Insert are not relevant and are shown as zero.
+For Insert operation, MAX represents total time to insert 1M entires into the cache and AVG represents the average time to insert one single entry. All other statistcs for Insert are not relevant and are set to zero.
 
 == Results
 |===
-|OPERATION TYPE|             MIN|          MAX|          AVG| MEDIAN|       MODE|         P99
-|SQL_QUERY_EXEC|             1.0000|      16.0000|       1.8739| 1.0000|       1.0000|       1.0000
-|SCAN_QUERY_EXEC|       530182.0000|  785735.0000|  561550.7678|  552705.0000|  543704.0000|  531308.3565
-|SQL_CURSOR_FETCH|           0.0000|       1.0000|       0.6717|   1.0000|       1.0000|       0.0000
-|SCAN_CURSOR_FETCH|          0.0000 |      8.0000 |       0.1411|       0.0000|       0.0000|       0.0000
-|SQL_INSERT|                 0.0000 | 929676278.00|     929.6762|    0.0|          0.0|          0.0
-|SCAN_INSERT|                0.0000|  812451365.00|     812.4514|    0.0|          0.0|          0.0
+|OPERATION_TYPE|    MIN|          MAX|          AVG| MEDIAN|       MODE|         P99
+|SQL_EXEC|        1.0000|       16.0000|     1.8739|     1.0000|     1.0000|     7.0000
+|SCAN_EXEC|  530182.0000|   785735.0000|561550.0000|552705.0000|543704.0000|667240.0000
+|SQL_FECTH|       0.0000|        1.0000|     0.6717|     1.0000|     1.0000|     1.0000
```

Comment:
Max = Medium = P99?

Reply:
Medium (1.0) is equal to P99 (1.0).  These statistics numbers are supposed be calculated automatically and easily.